js/main.js

```diff
@@ -0,0 +1,26 @@
+// Функция взята из источника https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+const randomInt = function getRandomIntInclusive(min, max) {
```

Comment:
Лучше использовать анонимную функцию в функциональном выражении, или давать тоже имя что и переменной с функцией, иначе, у переменной с функцией будет одно имя, а у самой функции другое, это не удобно, потому что в стек-трейсах будет имя функции.
```suggestion
const randomInt = function (min, max) {
```

```suggestion
const randomInt = function randomInt(min, max) {
```
```suggestion
const getRandomIntInclusive = function getRandomIntInclusive(min, max) {
```